Sekoia.io/sekoiaio/triggers/intelligence.py

```diff
@@ -100,10 +120,17 @@ def next_batch(self):
                 level="info",
             )
 
-            # Send objects as result of the trigger
-            self.send_event(
-                event_name=f"Sekoia.io feed - batch of {len(objects)} objects", event={"stix_objects": objects}
-            )
+            event_name = f"Sekoia.io feed - batch of {len(objects)} objects"
+            if self.to_file:
+                filepath = write("stix_objects.json", {"stix_objects": objects})
+                directory = filepath.parent.as_posix()
+
+                self.send_event(
+                    event_name=event_name, event={"file_path": filepath}, directory=directory, remove_directory=True
```

Comment:
The convention is to use the same name as for when it is not in a file with the `_path` suffix.

```suggestion
                    event_name=event_name, event={"stix_objects_path": filepath}, directory=directory, remove_directory=True
```

You must also update the trigger manifest to add this attribute as a result